src/sidebar.js

```diff
@@ -90,13 +90,28 @@ module.exports = {
             'guides/customize-identity-schema',
             {
               type: 'category',
-              label: 'Enabling Social Sign-in',
+              label: 'Social Sign-in',
```

Comment:
I think we can move this to it's own top level category, what do you think? Similar to `passwordless`

Reply:
Done!